src/BlockStorage/v2/Models/VolumeAttachment.php

```diff
@@ -0,0 +1,28 @@
+<?php declare (strict_types=1);
+namespace OpenStack\BlockStorage\v2\Models;
+
+use OpenCloud\Common\Resource\OperatorResource;
+
+/**
+ * @property \OpenStack\BlockStorage\v2\Api $api
+ */
+class VolumeAttachment extends OperatorResource
```

Comment:
if this resource doesn't have any API operations, it should extend from `AbstractResource`. `OperatorResource` provides a bunch of HTTP client helpers, which is irrelevant for models which are just value objects